src/main/java/me/moodcat/api/models/RoomModel.java

```diff
@@ -0,0 +1,8 @@
+package me.moodcat.api.models;
+
+/**
+ * @author Jan-Willem Gmelig Meyling
+ */
+public class RoomModel {
+
+}
```

Comment:
Is this for Jaap or is it pushed while not needed?
It seems just an empty class.